spirv_glsl.hpp

```diff
@@ -241,6 +501,9 @@ class CompilerGLSL : public Compiler
 	// - Images which are statically used at least once with Dref opcodes.
 	bool variable_is_depth_or_compare(VariableID id) const;
 
+	// TODO remove this function when all subgroup ops are supported (or make it always return true)
+	static bool is_supported_subgroup_op(spv::Op op);
```

Comment:
Does this have to be public? It does not seem to belong to the public API.

Reply:
Thought it might be useful for users. I can make it protected..